bam/inc/com/centreon/broker/bam/configuration/ba.hh

```diff
@@ -40,9 +40,23 @@ class ba {
   typedef enum {
     state_source_impact,
     state_source_best,
-    state_source_worst
+    state_source_worst,
```

Comment:
When calling this enum value, we have to write this right ?
`state_source.state_source_worst`

I would suggest this syntaxe instead :
`state_source.worst`

Reply:
The same here